VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddCommentFragment.java

```diff
@@ -149,7 +149,6 @@ public void onException(Exception ex) {
                 buttonAddComment.setEnabled(true);
             }
         };
-        createCommentTask.setTaskProcessCallback(null);
```

Comment:
Apparently git doesn't know that this change occurred in the `Task.java` class  \*ThugLife\* 😎